cylc/flow/scripts/lint.py

```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/env python3
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Cylc 728 looks through one or more folders for ``suite*.rc`` files and
```

Comment:
```suggestion
"""cylc lint [OPTIONS] ARGS

Look through one or more folders for ``suite*.rc`` files and
```

Reply:
Not addressed?

Reply:
There are still quite a few 728 references.